system/include/emscripten/fetch.h

```diff
@@ -184,6 +185,22 @@ EMSCRIPTEN_RESULT emscripten_fetch_wait(emscripten_fetch_t *fetch, double timeou
 // onerror() handler will be called in the calling thread before this function returns.
 EMSCRIPTEN_RESULT emscripten_fetch_close(emscripten_fetch_t *fetch);
 
+// Gets the size (in bytes) of the response headers as plain text.
+// Will only return a sensible result if the ready state is larger than or equal to 2.
```

Comment:
I would change that comment to something like:
Will return 0 if readyState < HEADERS_RECEIVED

The current comment makes it sound like this is undefined behaviour.

Reply:
Good idea :)